source/by-role/system-admin/performance-and-scaling.rst

```diff
@@ -1,5 +1,5 @@
 :is-up-to-date: True
-:last-updated: 4.1.2
+:last-updated: 5.0.0
```

Comment:
We still didn't split 4 and 5. I can't merge this yet. We either make this a draft, or we need to split, yet make 4 the default. I would rather draft this PR.